modules/local/create_process.nf

```diff
@@ -7,7 +7,7 @@ process CREATE_PROCESS {
     tuple val(target_id), path(test_json)
 
     output:
-    tuple val(target_id), path(test_json), path("dcqc-staged-*"), path("${test_json.baseName}.process.json")
+    tuple val(target_id), path(test_json), path("dcqc-staged-*/*"), path("${test_json.baseName}.process.json")
```

Comment:
Nice, does this work?

Reply:
discussed in slack, but yes, this does work coupled with the changes in the `py-dcqc` pr which passes the file name to the command, rather than a relative path which includes the staging folder.